frontend/src/components/dashboard/Dashboard.js

```diff
@@ -161,16 +164,24 @@ const DropDownList = ({ buttonRef, handleOpen, handleClose, items, open }) => {
   const classes = useStyles();
   const { startLoading } = useContext(UserContext);
 
-  const handleClick = () => {
-    startLoading();
+  const handleClick = (onClick) => {
+    if (onClick) {
+      onClick();
+    } else {
+      startLoading();
+    }
   };
 
   return (
     <Popper open={open} anchorEl={buttonRef.current}>
       <Paper onMouseEnter={handleOpen} onMouseLeave={handleClose} className={classes.menu}>
         <MenuList>
           {items?.map((item) => (
-            <Link key={item.url_slug} href={item.url_slug} onClick={handleClick}>
+            <Link
+              key={item.url_slug}
+              href={item.url_slug}
```

Comment:
The href here needs to be removed if item.url_slug is empty. Otherwise, it will open the idea popup for a second and then just refresh the current page

Reply:
For the idea popup, the href is empty string. I tested it and it does not redirect you anywhere